samples/open-telemetry/logging/Core_8/GenericHost/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+internal class Program
+{
+    public static void Main(string[] args)
+    {
+        CreateHostBuilder(args).Build().Run();
+    }
+
+    private static IHostBuilder CreateHostBuilder(string[] args)
+    {
+        var builder = Host.CreateDefaultBuilder(args);
+        builder.UseWindowsService();
+
+        #region generic-host-nservicebus
+
+        builder.UseNServiceBus(ctx =>
+        {
+            var endpointConfiguration = new EndpointConfiguration("Samples.Hosting.GenericHost");
+            endpointConfiguration.UseTransport(new LearningTransport());
+
+            endpointConfiguration.DefineCriticalErrorAction(OnCriticalError);
```

Comment:
is that required/useful for this sample?

Reply:
Not specifically for the sample, but it seems like a good enough practice to not remove it? As stated before, I based this off another sample